x-pack/plugin/mapper-constant-keyword/src/main/java/module-info.java

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/** Elasticsearch X-Pack Constant Keyword Mapper Plugin. */
+module org.elasticsearch.xpack.constantkeyword {
```

Comment:
you can omit the "xpack" portion of all of these module names, xpack at this point is a source code division, but they are all just ES modules.

Reply:
Ah got it (makes sense to me going w/o xpack for long term), I will!
Thank you for your prompt check.

Reply:
Addressed with [`166d4be` commit](https://github.com/elastic/elasticsearch/pull/131486/commits/166d4be3caee47d58b80cd8c9deb7bd36576f3a4)